modules/sovrnBidAdapter.js

```diff
@@ -268,11 +264,16 @@ export const spec = {
 
 function _buildVideoRequestObj(bid) {
   const videoObj = {}
+  const bidSizes = deepAccess(bid, 'sizes')
   const videoAdUnitParams = deepAccess(bid, 'mediaTypes.video', {})
   const videoBidderParams = deepAccess(bid, 'params.video', {})
   const computedParams = {}
-
-  if (Array.isArray(videoAdUnitParams.playerSize)) {
+  
+  if (bidSizes) {
```

Comment:
This is the changes for the size stuff, no clue if this one is right, might need more clarification.